hw/xilinx/Makefile

```diff
@@ -45,13 +45,19 @@ endif
 SYNTH_STRATEGY ?= Flow_RuntimeOptimized
 IMPL_STRATEGY ?= Flow_RuntimeOptimized
 
+# Implementation artifacts
+XILINX_BITSTREAM   ?= ${XILINX_PROJECT_BUILD_DIR}/${XILINX_PROJECT_NAME}.runs/impl_1/${XILINX_PROJECT_NAME}.bit
+XILINX_PROBE_LTX ?= ${XILINX_PROJECT_BUILD_DIR}/${XILINX_PROJECT_NAME}.runs/impl_1/${XILINX_PROJECT_NAME}.ltx
+
+# Whether to use ILA probes (0|1)
+XILINX_ILA ?= 1
```

Comment:
It could be good to add this to a README?

Reply:
good idea